cli.js

```diff
@@ -100,15 +100,14 @@ var api = new Api(cli.input.length ? cli.input : arrify(conf.files), {
 	cacheEnabled: cli.flags.cache !== false
 });
 
-var logger = new Logger();
-logger.api = api;
+var logger;
 
 if (cli.flags.tap) {
-	logger.use(tapReporter());
+	logger = new Logger(tapReporter(api));
```

Comment:
I think this would be cleaner:

```js
var reporter;

if (cli.flags.tap) {
  reporter = new TapReporter();
}

var logger = new Logger(reporter);
```